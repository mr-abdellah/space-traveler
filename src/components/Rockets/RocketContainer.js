import React from 'react';

export default function RocketContainer() {
  return (
    <div>RocketContainer</div>
  );
}
