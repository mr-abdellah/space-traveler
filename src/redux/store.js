import { configureStore } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import rocketReducer from './rockets';
import missionsReducer from './missions';

const store = configureStore({
  reducer: {
    rockets: rocketReducer,
    missions: missionsReducer,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});

export default store;
