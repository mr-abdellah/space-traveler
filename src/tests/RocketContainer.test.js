import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import RocketContainer from '../components/Rockets/RocketContainer';

it('Test Rocket component:', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <RocketContainer />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
