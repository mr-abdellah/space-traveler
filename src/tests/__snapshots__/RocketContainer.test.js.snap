// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Rocket component: 1`] = `null`;
